Build.md

```diff
@@ -1,6 +1,6 @@
 # Build & Publish #
 
-The information below describes the steps that the default build script (Build-All.ps1) performs.
+The information below describes the steps that the default build script (build.ps1) performs.
```

Comment:
since there's only one script now please give a TLDR instruction (no more than 3 lines perhaps) on how to build the client without any thoughts. Can be placed either in this file or in readme

Reply:
I am against "one script" approach, please provide the front-end script that just launches the regular configs without any extra actions.

Reply:
> I am against "one script" approach, please provide the front-end script that just launches the regular configs without any extra actions.

On Windows, double-clicking opens it with a text editor rather than executing the script.

The default behavior of Windows PowerShell is to disable any unsigned script execution, and that "Run with PowerShell" context menu is affected by this.

PowerShell Core, which allows any script to be executed by default, does not register a context menu like "Run with PowerShell Core".

Running ps1 scripts like cmd scripts is difficult. So why do you need all these front-end scripts?

If you do want front-end scripts, I'll write a tool to create shortcuts.
These shortcuts will point to powershell.exe or pwsh.exe, passing the full path to the compiled script and game variables as its command line arguments.

Reply:
provide some bat files invoking the ps1 script while bypassing the policy

Reply:
> provide some bat files invoking the ps1 script while bypassing the policy

bat scripts don't do the job of bypassing PowerShell restrictions.
So I have to write a bat script that uses pwsh to run it?

![image](https://github.com/CnCNet/xna-cncnet-client/assets/42184238/8355fbf9-2b71-4ce4-aca6-bc0933b017ae)

I need to gather some information such as
Do developers prefer Windows PowerShell 5.1 or PowerShell Core 7.3?

Reply:
@frg2089 https://stackoverflow.com/a/42898542 When using the powershell.exe command, note that the -executionpolicy parameter must be put before the -File parameter or it won't work.